apps/mudita-center-e2e/src/specs/settings/backup-location.e2e.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+import { sleep } from "../../helpers"
+import HomePage from "../../page-objects/home.page"
+import SettingsPage from "../../page-objects/settings.page"
+import NavigationTabs from "../../page-objects/tabs.page"
+
+describe("Open Settings and change backup location", () => {
+  before(async function () {
+    const notNowButton = await HomePage.notNowButton
+    await notNowButton.waitForDisplayed()
+    await notNowButton.click()
+  })
+  it("Verify default settings", async () => {
+    const settingsTab = await NavigationTabs.settingsTab
+    await settingsTab.waitForDisplayed()
+    await settingsTab.click()
+
+    const aboutTab = await SettingsPage.aboutTab
+    await aboutTab.waitForDisplayed()
+
+    const changeLocationButton = await SettingsPage.changeLocationButton
+    await changeLocationButton.waitForClickable()
```

Comment:
test fails on  2.2.8-dev.580 (linux) `Error: element ("[data-testid="settings-backup-change-location-button"]") still not clickable after 6000ms`

Reply:
passed on 2.2.8-dev.581 (linux)